reporter/build.gradle.kts

```diff
@@ -64,12 +65,16 @@ repositories {
 dependencies {
     api(project(":model"))
 
+    api("com.squareup.retrofit2:retrofit:$retrofitVersion")
```

Comment:
Why is this not an `implementation` dependency?

Reply:
Because the public `POST` / `GET` functions [return a Call](https://github.com/oss-review-toolkit/ort/pull/2467/files#diff-95a39e715a04fb84d1c292fdf3c9fb68R177) which is an OkHttp type.

Reply:
But the external API of the reporter are the reporter classes, not the REST service. What about making the service internal instead?

Reply:
Yes, that would be feasible. Please do so, @julscha, and `typealias ProjectAcl` then needs to be changed to `internal typealias ProjectAcl`, too.